### PR TITLE
Fix #518: Fix overflow of probe name

### LINKED ISSUE #518: Probe name goes beyond the page
When the probe name is too long, it goes beyond the page like [this](https://telemetry.mozilla.org/probe-dictionary/?detailView=histogram%2FCACHE_SERVICE_LOCK_WAIT_MAINTHREAD_NSCACHEENTRYDESCRIPTOR_GETCLIENTID). It should be displayed in the page limit...

### PR DESCRIPTION
**Pull request by Insiyaa**
Large probe name lead the user to scroll beyond the page. Hence probe name should be displayed in more than one line.
screenshot-2018-4-9 firefox probe dictionary - find probes

### CODE REVIEW

**Comment by georgf** [comment]
Do you have a live link for this change @Insiyaa? Thanks.

**Comment by Insiyaa** [comment]
https://insiyaa.github.io/telemetry-dashboard/probe-dictionary/ here's the link

File: probe-dictionary/explorer.css [hunk 1]
```diff
@@ -155,3 +155,7 @@ th {
     top: -1px;
     background-color: #ECECEC;
 }
+
+#detail-probe-name {
+    overflow-wrap: break-word;
+}
```
Comment (georgf) [inline probe-dictionary/explorer.css]: A very small thing: This is missing a linebreak. Usually we have an empty newline at the end of files, see [e.g. here](https://stackoverflow.com/a/729795/).

**Review by georgf — COMMENTED** [review]
This works well, thanks!
I only have a small comment below, then we can land this.

File: probe-dictionary/explorer.css (hunk 1 above)
Comment (Insiyaa) [reply probe-dictionary/explorer.css]: Yes done!

Comment (Insiyaa) [reply probe-dictionary/explorer.css]: Thank you!

**Review by georgf — APPROVED** [review]
Thanks Insiyaa, that looks good!
I like how simple the change turned out.

**Comment by Insiyaa** [comment]
Thank You! :)

**Pull request merged**